lib/WebSocket.js

```
@@ -98,10 +98,15 @@ class WebSocket extends EventEmitter {
   }
 
   set binaryType (type) {
-    if (type === 'arraybuffer' || type === 'nodebuffer') {
-      this._binaryType = type;
-    } else {
-      throw new SyntaxError('unsupported binaryType: must be either "nodebuffer" or "arraybuffer"');
+    if (constants.BINARY_TYPES.indexOf(type) < 0) {
+      throw new SyntaxError('unsupported binaryType: must be either "' +
```

Firefox doesn't warn, it just fails silently. What does WHATWG spec say?